TicTacToeApp/Game.swift

```diff
@@ -0,0 +1,36 @@
+
+public class Game {
+
+  private var board: Board
+  private var currentPlayer: PlayerMark
+
+  public init() {
+    board = Board()
+    currentPlayer = PlayerMark.X
+  }
+
+  public func makeMove(boardSpace: Int) {
+    board = board.markSpace(boardSpace, mark: currentPlayer)
+    currentPlayer = otherPlayer()
+  }
+
+  public func getCurrentPlayer() -> PlayerMark {
```

Comment:
I thought you could do it in a way such that the getter and setter have different private/public scoping, but I could be wrong.

Let's discuss in person.